parse/NamedValueRefParser.cpp

```diff
@@ -32,8 +32,8 @@ namespace parse {
                           bool& pass)
     {
         ErrorLogger() << "Registering from named_values.focs.txt : " << name << " ! ValueRef<" << typeid(T).name() << ">";
-        //named_refs.emplace(name, ref_envelope.OpenEnvelope(pass)); // does not know how to handle different types
-        // Note: using ::RegisterValueRef(name, ref_envelope.OpenEnvelope(pass)) circumvents the Pending mechanism
+        // Note: Other parsers might also register value refs, so
+        //       we do not use the named_refs and as the normal pending mechanism does not suffice.
```

Comment:
"we do not use the named_refs"?

Reply:
i meant we do not use the named_refs parameter (anymore) to collect named refs.